Automattic/remote-data-blocks
# add airtable rich text support

# Description

> [!NOTE]
> JIRA: [CAFE-1090](https://vipjira.atlassian.net/browse/CAFE-1090)

- Adds a new type `markdown` in output query mappings. The values are assumed as markdown string and is converted to HTML.
- Adds support for Airtable rich text fields by mapping them to the newly added 'markdown' type. These changes allow rich text content from Airtable to be correctly parsed and displayed as HTML.


## Testing

To test these changes:

1. Start the local development server using instructions given [here](https://github.com/Automattic/remote-data-blocks/blob/trunk/docs/local-development.md)
2. Setup an Airtable Base with Table having a field with type "Long Text" and rich text enabled.
3. Open the WP-Admin and go to Settings > Remote Data Blocks and add new Airtable data source by selecting the above Airtable Base/Table combination and select the field in field selection input.
4. Verify that the rich text field is displayed correctly in the block.
